src/components/Navbar/index.tsx

```diff
@@ -1,12 +1,5 @@
-import useMobile from 'hooks/Mobile'
 import DesktopNavigation from './DeskTopNavigation'
-import MobileCollapse from './Mobile/MobileCollapse'
 
 export default function Navbar() {
-  const isMobile = useMobile()
-  const desktop = !isMobile
-
-  if (isMobile) return <MobileCollapse />
-
-  return <DesktopNavigation desktop={desktop} />
+  return <DesktopNavigation />
```

Comment:
does this component serve any purpose anymore if it just returns another component?

Reply:
Nah, I can get rid of it in another PR that removes the component.